### PR TITLE
Update project.md

### LINKED ISSUE #282: Rethink the scope of the project
There was a feeling across both London and Nazareth that the project was very similar to the project done the previous week. There's an agreement that this could be beneficial to consolidate material but at the same time that could also be boring to some people. We should think how to reframe the project to maybe give people more options.

### PR DESCRIPTION
**Pull request by majakudlicka**
We have reframed the project to include user input and server validation.
We have added the comment to host project in students own repos rather than FAC11.
We have made the scope of project more flexible.

fixes #282

### CODE REVIEW

**Comment by bradreeder** [comment]
Thanks for this @majakudlicka . I will leave this open for mentors from Nazareth to review.

Also please remember to reference the issue you're working against when making PRs. I've just added it to the description. 👍

File: coursebook/week-5/project.md [hunk 1]
```diff
@@ -33,10 +37,12 @@ This news feed could be created with data provided via the [TFL API](https://api
 
 8) Consider a good server file structure based on what we have discussed over the week.
 
-9) Employ continuous intergration on your project with Travis or a similar tool.
+9) Employ continuous intergration on your project with Travis or a similar tool. (If you decide to use Travis, we strongly recommend that you host this project in your own repo rather than in FAC11 to avoid all builds getting queued together)
```
Comment (jsms90) [inline coursebook/week-5/project.md]: Please don't explicitly mention a particular cohort number :+1:

File: coursebook/week-5/project.md [hunk 2]
```diff
@@ -33,10 +37,12 @@ This news feed could be created with data provided via the [TFL API](https://api
...
 
 10) Use CodeCov or a similar tool for test coverage.
 
 11) Display continuous intergration and code coverage badges on your project README. 
 
 12) Ensure that errors are handled, if for example a user attempts to make a request to a non-existent route to your server, provide the user with a custom response.    
+
+13) Include user input field on your web app and include server-side validation to protect your server from potentially malicious user input.
```
Comment (jsms90) [inline coursebook/week-5/project.md]: Include "a" user input field

File: coursebook/week-5/project.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
-Create a website that will provide a news feed, with the content sourced via an API. 
+Create a website that will include user input and a feed, with the content sourced via an API. 
 
 ### Example:
 
-User Story:
+Example User Stories:
 'As a student at F&C I want to know all the train departure times from Bethnal Green tube station, so that I can get home in time for dinner'.
 
+'I want to input which direction of travel I am interested in'.
```
Comment (jsms90) [inline coursebook/week-5/project.md]: Can we frame this user story in the regular format please?

**As a**
**I want to**
**So that**

**Comment by majakudlicka** [comment]
Also, I know we discussed contacting Travis - but it just struck me that perhaps it would look a bit more professional if the query was sent from Founders & Coders email rather than my private email. Thoughts?

**Comment by jsms90** [comment]
@majakudlicka Agreed, but it seems like the way that you get in touch with Travis is by logging into their website with your Travis account. Resolving this problem is pretty urgent now, considering the fact that students will see this file in less than a week, so I've just sent them a message.

I feel very uncomfortable about this line
> If you decide to use Travis, we strongly recommend that you host this project in your own repo rather than in your cohort's FAC repository to avoid all builds getting queued together

but I get that we don't have a good solution at the moment for introducing Continuous Integration & enabling students to continue using the shared github org. So I'm happy to approve for now, but have opened this as a separate issue

**Pull request merged**